test/Empty.csproj

```diff
@@ -0,0 +1,6 @@
+<!-- This is an empty *.csproj so that dependabot can bump dependencies in Directory.Build.props. -->
```

Comment:
Could you please open issue on dependabot side to cover such scenario?
I think that you have good enough description in our PR to describe how it should work.

Reply:
Addressed https://github.com/dependabot/dependabot-core/issues/1182#issuecomment-1138312743